_data/osdf_namespace_metadata/ndp-burnpro3d.yaml

```
@@ -18,7 +18,7 @@ namespace:
   - "/ndp/burnpro3d"
 thirtyDayReads: 316964599
 oneYearReads: 9337358188
-publicObject: /ndp/burnpro3d/TLS_scans.zip # Optional namespace path to a public object
+publicObject: /ndp/burnpro3d/YosemiteBurnExample/ # Optional namespace path to a public object
```

```suggestion
publicObject: /ndp/burnpro3d/YosemiteBurnExample/run_bc4c3bc3-68a7-4440-9b07-6f79f4577420_quicfire.zarr # Optional namespace path to a public object
```

The public object should be something that is downloadable via an "object get" call with no flags (such as -r for recursive), not a directory. It's really as an example download, so it's okay if it isn't the whole dataset.